src/wallet/rpcwallet.cpp

```diff
@@ -1670,11 +1671,13 @@ static UniValue gettransaction(const JSONRPCRequest& request)
             "    ,...\n"
             "  ],\n"
             "  \"hex\" : \"data\"         (string) Raw data for transaction\n"
+            "  \"decoded\" : \"transaction\"         (json object) Optional, the decoded transaction\n"
```

Comment:
Remove `\"transaction\"` as the value is not a string. Maybe mention that it's the same result as `decoderawtransaction`?

Reply:
done